### DISCUSSION TITLE
Should I use `markuplint:recommended-vue` rather than `markuplint:recommended` for Vue project?

### DISCUSSION THREAD

**Discussion by munierujp** [markuplint/markuplint#777] [Q&A]
When I select Vue as the template engine at the prompt after running `npx markuplint -init`, the generated configuration file still uses `markuplint:recommended` (NOT `markuplint:recommended-vue`):

```json
{
  "specs": {
    "\\.vue$": "@markuplint/vue-spec"
  },
  "parser": {
    "\\.vue$": "@markuplint/vue-parser"
  },
  "extends": [
    "markuplint:recommended"
  ]
}
```

Why is `markuplint:recommended-vue` not used? In what cases should `markuplint:recommended-vue` be used?

---

Moved from issue #774

**Comment by trapegen** [comment]
When you select Vue as the template engine with npx markuplint -init, it generates a configuration file with "markuplint:recommended" as an extension. This is because "markuplint:recommended" includes all the core rules of Markuplint that apply to HTML, and "markuplint:recommended-vue" includes additional rules that are specific to Vue template syntax.

If you are using Vue templates, it's recommended to extend "markuplint:recommended-vue" instead of "markuplint:recommended". This will ensure that all the rules that are relevant to Vue templates are enabled. However, if you are only working with HTML files, it's not necessary to extend "markuplint:recommended-vue".

You can change the configuration file to extend "markuplint:recommended-vue" instead of "markuplint:recommended" to enable Vue-specific rules.
